### PR TITLE
Workaround issue with `hasOwnProperty`

### PR DESCRIPTION
**Pull request by OliverJAsh**
Re. https://github.com/gcanti/fp-ts/issues/1249#issuecomment-820232156

### CODE REVIEW

File: src/internal.ts [hunk 1]
```diff
@@ -24,8 +24,17 @@ export const isLeft = <E, A>(ma: Either<E, A>): ma is Left<E> => ma._tag === 'Le
 // Record
 // -------------------------------------------------------------------------------------
 
-/** @internal */
-export const hasOwnProperty = Object.prototype.hasOwnProperty
```
Comment (gcanti) [inline src/internal.ts:28]: @OliverJAsh we can just change the name to `has` here

**Review by gcanti — CHANGES_REQUESTED** [review]

File: src/internal.ts (hunk 1 above)
Comment (OliverJAsh) [reply src/internal.ts:28]: Good idea, done!

**Review by gcanti — APPROVED** [review]

**Comment by OliverJAsh** [comment]
Thanks! I see a 2.10.1 tag has been pushed to GitHub but the release doesn't seem to exist on npm yet?

**Comment by gcanti** [comment]
@OliverJAsh should be a matter of seconds, let me know if it's ok

**Pull request merged**